src/containers/AccessPointDetails/index.js

```diff
@@ -43,6 +43,7 @@ const AccessPointDetails = ({
   extraTabs,
   extraGeneralCards,
   showStatusAlarms,
+  extraIcon,
```

Comment:
rename this prop as `avatar` this isn't 'extra' it's an override prop not adding